charts/bitbucket-runner-autoscaler/templates/runnernamespace.yaml

```diff
@@ -1,12 +1,6 @@
-{{- $allNamespaces := (list) }}
-{{- range .Values.runner.config.groups }}
-  {{- $allNamespaces = append $allNamespaces .namespace }}
-{{- end }}
-{{- range $namespace := $allNamespaces | uniq }}
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: {{ $namespace }}
+  name: bitbucket-runner
```

Comment:
I think the name should include release name, otherwise when installing another runner stack it will fail